cmd/kedge/main.go

```diff
@@ -69,6 +69,24 @@ var (
 			"director & backendpool routings. It will update them directly into into flagz value, so you can see the current routings anytime in debug/flagz")
 )
 
+const (
+	grpcWebContentTypePrefix = "application/grpc-web"
+	grpcContentTypePrefix    = "application/grpc"
+)
+
+func isGRPCReq(header http.Header) bool {
+	// Do not treat "grpc-web" request as pure gRPC. It is pure HTTP/2 at this point.
+	if strings.HasPrefix(header.Get("content-type"), grpcWebContentTypePrefix) {
+		return false
+	}
+
+	if strings.HasPrefix(header.Get("content-type"), grpcContentTypePrefix) {
```

Comment:
You don't have to use `const` if you use the const string only once. Just put it here directly and it will be easier to read and maintain.

Reply:
If you say so, why not

Reply:
We need to minimize amount of LOC to minimum. Less LOC = better readability due to less indirection and less bugs. We can increase LOC to make programs DRY (repeating yourself is worse than more LOC).